Makefile

```diff
@@ -1,11 +1,20 @@
+export CGO_ENABLED:=0
+
 VERSION=$(shell ./scripts/git-version)
+GOPATH_BIN:=$(shell echo ${GOPATH} | awk 'BEGIN { FS = ":" }; { print $1 }')/bin
 
 .PHONY: all
 all: build
 
 .PHONY: build
-build:
-	@go install -v github.com/coreos/terraform-provider-matchbox
+build: bin/terraform-provider-matchbox
+
+bin/%:
+	@go build -o bin/$* -v github.com/coreos/terraform-provider-matchbox
```

Comment:
hm? Running `make build` creates `bin/terraform-provider-matchbox` for local development or install. And the release outputs for linux and darwin, they do the build separately into the _output folder to create tarballs